hazelcast-enterprise/Dockerfile

```diff
@@ -26,6 +26,10 @@ ARG EUREKA_PLUGIN_URLS='https://repo1.maven.org/maven2/com/hazelcast/hazelcast-e
 
 # Runtime constants / variables
 ENV HZ_HOME="${HZ_HOME}" \
+    HZ_CP_MOUNT="${HZ_HOME}/external" \
```

Comment:
How about making these env similar to `hazelcast-oss`? Like not to specify `10001`?

Reply:
Done 👍